bigquery_etl/format_sql/format.py

```diff
@@ -52,7 +54,13 @@ def format(paths, check=False):
         for path in sql_files:
             with open(path) as fp:
                 query = fp.read()
```

Comment:
nit: since using a path object downstream anyway
```suggestion
            query_path = Path(path):
            query = query_path.read_text()
```
or `query_path.read_text()` on line 61